package.json

```diff
@@ -26,7 +26,8 @@
     "dependencies": {
         "@emotion/core": "^10.0.5",
         "@guardian/consent-management-platform": "^2.0.10",
-        "@guardian/slot-machine-client": "^0.1.57",
+        "@guardian/dotcom-rendering": "file:",
```

Comment:
I am a little confused as what is going on here?

Reply:
Glad you spotted that. It's gone now, but that's a little yarn oddity when you add a local package by running `yarn add file: ../slot-machine-client` instead of `yarn add @guardian/slot-machine-client`. We sometimes do that to use a npm package we have locally, so we don't need to publish to npm every time we wanna test it on DCR. For some reason, every time we do this it creates that off entry in the `package.json` that needs to be manually deleted. @liywjl